pyxem/generators/indexation_generator.py

```diff
@@ -45,6 +48,29 @@
 
 import lmfit
 
+def get_fourier_transform(template_coordinates, template_intensities, shape, fsize):
+    template = np.zeros((shape))
+    template[template_coordinates[:, 1], template_coordinates[:, 0]] = template_intensities[:]
+    template_FT = np.fft.fftshift(np.fft.fftn(template,fsize))
```

Comment:
Does it still work if you use `np.fft.rfftn()`? You get a 2x speed-up using that (assuming the data supports this assumption).